lib/index.js

```diff
@@ -3,6 +3,8 @@
 import LinterUI from './main'
 import type Intentions from './intentions'
 
+const idleCallbacks = new Set()
```

Comment:
This one is package level, the one in main.js is specific to a `LinterUI` instance, correct?